src/internal/observable/concat.ts

```diff
@@ -75,7 +75,10 @@ export function concat<R>(...observables: (ObservableInput<any> | SchedulerLike)
  * const timer1 = interval(1000).pipe(take(10));
  * const timer2 = interval(2000).pipe(take(6));
  * const timer3 = interval(500).pipe(take(10));
- * const result = concat([timer1, timer2, timer3]); // note that array is passed
+ *
+ * // an array itself being considered an observable source,
+ * // use the spread operator to observe its values instead of itself
+ * const result = concat(...[timer1, timer2, timer3]); // note that array is spread
```

Comment:
Honestly, this makes me wonder if there was a regression at some point where we used to take a single array. 🤔 

Either way, for now, it's probably best just to pass them in without spreading them from an array, because it doesn't make sense.